### PR TITLE
chore: add script for running WPT on all modes

### PR DESCRIPTION
**Pull request by jrandolf**
This allows maintainers to update expectations across all modes without executing each mode manually.

### CODE REVIEW

**Review by OrKoN — APPROVED** [review]

**Comment by thiagowfx** [comment]
Please wait for my review on this one. I have a local script that does something similar and we could benefit from merging it with yours

**Comment by thiagowfx** [comment]
PTAL

**Review by thiagowfx — APPROVED** [review]

**Comment by thiagowfx** [comment]
Ah, wait. We need a small change to make it work without arguments too.

File: runWPTAll.sh [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+set -uo pipefail
+
+trap ctrl_c INT
+ctrl_c() {
+    exit 1
+}
+
+export LOG_FILE="logs/$(basename "$0").log"
+export UPDATE_EXPECTATIONS="${UPDATE_EXPECTATIONS:-false}"
```
Comment (Lightning00Blade) [inline runWPTAll.sh:12]: I have needed to use it yet so, @jrandolf do you think setting update expectation default to true is more in line with what this script will be used for?

Comment (jrandolf) [reply runWPTAll.sh:12]: @thiagowfx

Comment (thiagowfx) [reply runWPTAll.sh]: I tend to use both values. `false` during development, `true` when I am ready to regenerate the expectations. We can set this to `false`.

**Pull request merged**